bson/src/main/org/bson/types/ObjectId.java

```diff
@@ -351,12 +363,29 @@ public String toString() {
         return toHexString();
     }
 
-    // see https://docs.oracle.com/javase/6/docs/platform/serialization/spec/output.html
+    /**
+     * Write the replacement object.
+     *
+     * <p>
+     * See https://docs.oracle.com/javase/6/docs/platform/serialization/spec/output.html
```

Comment:
[optional, ignore if consider to be too much of a hassle]

Let's update to https://docs.oracle.com/en/java/javase/17/docs/specs/serialization/output.html#the-writereplace-method. This comment is applicable to all other similar changes.

Reply:
Too much of a hassle :)